### PR TITLE
Add YAESU FT-600 (System) Model [ALPHA]

### PR DESCRIPTION
**Pull request by mrmoonlv**

### CODE REVIEW

File: include/hamlib/riglist.h [hunk 1]
```diff
@@ -499,7 +500,7 @@
 
 
 /*
- * VEB Funkwerk K�penick RFT
+ * VEB Funkwerk K�penick RFT
```
Comment (mrmoonlv) [inline include/hamlib/riglist.h]: Some typo garbage. Please remove this line.

**Comment by N0NB** [comment]
* On 2019 08 Mar 12:48 -0600, Kārlis Millers wrote:
> mrmoonlv commented on this pull request.
> 
> 
> 
> > @@ -499,7 +500,7 @@
>  
>  
>  /*
> - * VEB Funkwerk K�penick RFT
> + * VEB Funkwerk K�penick RFT
> 
> Some typo garbage. Please remove this line.

You can add a commit to the PR to fix it as you need to.

73, Nate

-- 

"The optimist proclaims that we live in the best of all
possible worlds.  The pessimist fears this is true."

Web: http://www.n0nb.us  GPG key: D55A8819  GitHub: N0NB

**Comment by mrmoonlv** [comment]
Thanks, Nate.
I will continue development of this model. Need some testing on wsjtx and fldigi. 
73, Karlis.

**Comment by mrmoonlv** [comment]
@N0NB . Updated..

**Comment by N0NB** [comment]
* On 2019 15 Mar 06:37 -0500, Kārlis Millers wrote:
> @N0NB . Updated..
> 

Hi Kārlis,

I am seeing the following warnings with GCC 8.3.0 on Debian Testing:

  CC       ft600.lo
In file included from ../../hamlib/yaesu/ft600.c:39:
../../hamlib/yaesu/ft600.h:133:12: warning: ‘ft600_set_vfo’ declared ‘static’ but never defined [-Wunused-function]
 static int ft600_set_vfo(RIG *rig, vfo_t vfo);
            ^~~~~~~~~~~~~
../../hamlib/yaesu/ft600.h:137:12: warning: ‘ft600_get_ptt’ declared ‘static’ but never defined [-Wunused-function]
 static int ft600_get_ptt(RIG *rig, vfo_t vfo, ptt_t *ptt);
            ^~~~~~~~~~~~~
../../hamlib/yaesu/ft600.c: In function ‘ft600_get_mode’:
../../hamlib/yaesu/ft600.c:448:30: warning: array subscript 6 is above array bounds of ‘unsigned char[4]’ [-Warray-bounds]
     switch( priv->status.freq[6] ) {
             ~~~~~~~~~~~~~~~~~^~~

Thanks for looking into these.

73, Nate

-- 

"The optimist proclaims that we live in the best of all
possible worlds.  The pessimist fears this is true."

Web: http://www.n0nb.us  GPG key: D55A8819  GitHub: N0NB

**Comment by mrmoonlv** [comment]
@N0NB fixed.
73, Karlis.

**Comment by N0NB** [comment]
Applied and pushed.  Thanks, Karlis.

73, Nate

-- 

"The optimist proclaims that we live in the best of all
possible worlds.  The pessimist fears this is true."

Web: http://www.n0nb.us  GPG key: D55A8819  GitHub: N0NB

**Comment by k6dvd** [comment]
Glad to see that the FT-600 will be added to hamlib.  I currently use my FT-600 connected to a RPi3b+ running wsjtx 2.0.1 by selecting FT840 for the CAT control and it seems to work.  The usb to radio cable I use was also intended for the FT-840.
-Dave

**Pull request merged**